ruby/hashes.rb

```diff
@@ -0,0 +1,72 @@
+=begin 
+-build hash with empty keys for name, age, number of children
+decor theme, allergies, payment method
+-prompt user for imput
+	-enter user data into proper key
+-print hash
+-ask if data is correct
+	- if correct user enters "none" to procede
+	- if key needs revision user enters key 
+		-updates key value
+-print hash
+=end
+
+client ={
+	name: nil, 
+	age: nil,
+	num_child: nil,
+	theme: nil,
+	allergies: ["none"],
+	payment_method: nil
+}
+	def allergies_question 
+		user_answer="nil"
```

Comment:
There are a lot of extra tabs here.  One tab in per nested level is the best practice.